src/openshift/component.ts

```diff
@@ -292,10 +292,9 @@ export class Component extends OpenShiftItem {
                         case 'Undeploy':
                             await Component.undeploy(component);
                             return null;
-                            break;
                         case 'Help':
                             open('https://github.com/redhat-developer/vscode-openshift-tools/wiki/Migration-to-v0.1.0');
-                            break;
+                            return null;
```

Comment:
This change makes message disappear after 'Help' selected. The idea to keep it coming back after 'Help' selected. This way when you done reading documentation you can switch back to vscode and continue the workflow.